yt_dlp/extractor/bandcamp.py

```diff
@@ -429,3 +421,116 @@ def _real_extract(self, url):
                 ie=self._TYPE_IE_DICT[item[1]])
             for item in items]
         return self.playlist_result(entries, id)
+
+
+class BandcampUserIE(InfoExtractor):
+    IE_NAME = "Bandcamp:user"
+    _VALID_URL = r"https?://(?:(?P<id>[^.]+)\.)?bandcamp\.com"
+
+    _TESTS = [{
+        # Type 1 Bandcamp user page.
+        "url": "https://adrianvonziegler.bandcamp.com",
+        "info_dict": {
+            "id": "adrianvonziegler",
+            "title": "Discography of adrianvonziegler",
+        },
+        "playlist_mincount": 23,
+    }, {
+        # Bandcamp user page with only one album
+        "url": "http://dotscale.bandcamp.com",
+        "info_dict": {"id": "dotscale", "title": "Discography of dotscale"},
+        "playlist_count": 1,
+    }, {
+        # Type 2 Bandcamp user page.
+        "url": "https://nightcallofficial.bandcamp.com",
+        "info_dict": {
+            "id": "nightcallofficial",
+            "title": "Discography of nightcallofficial",
+        },
+        "playlist_count": 4,
+    }, {
+        'url': 'https://steviasphere.bandcamp.com/music',
+        'playlist_mincount': 47,
+        'info_dict': {
+            'id': 'steviasphere',
+        },
+    }, {
+        'url': 'https://coldworldofficial.bandcamp.com/music',
+        'playlist_mincount': 10,
+        'info_dict': {
+            'id': 'coldworldofficial',
+        },
+    }, {
+        'url': 'https://nuclearwarnowproductions.bandcamp.com/music',
+        'playlist_mincount': 399,
+        'info_dict': {
+            'id': 'nuclearwarnowproductions',
+        },
+    },
+    ]
+
+    @classmethod
+    def suitable(cls, url):
+        return (
+            False
+            if BandcampAlbumIE.suitable(url)
+            or BandcampIE.suitable(url)
+            or BandcampWeeklyIE.suitable(url)
+            else super(BandcampUserIE, cls).suitable(url)
+        )
+
+    def _real_extract(self, url):
+        uploader = self._match_id(url)
+        webpage = self._download_webpage(url, uploader)
+        entries = []
+
+        # Bandcamp User type 1 page
+        discography_data = re.findall(
+            r'<li data-item-id="([^"]+)[^>]+>\s*<a href="(/[^/]+/[^/"]+)">',
+            webpage,
+            re.MULTILINE,
+        )
+
+        if len(discography_data) > 0:
+            for match in discography_data:
+                element_id = match[0]
+                element_url = match[1]
+                if element_url.split("/")[1] == "album":
+                    ie = BandcampAlbumIE.ie_key()
+                else:
+                    ie = BandcampIE.ie_key()
+
+                entries.append(
+                    self.url_result(
+                        urljoin(url, element_url),
+                        ie=ie,
+                        video_id=element_id,
+                        video_title=element_url.split("/")[2],
+                    )
+                )
```

Comment:
```suggestion
            entries = [self.url_result(urljoin(url, element_url)) for element_id, element_url in discography_data]
```
Return id and title only if they are actually correct

Reply:
Fixed in 1fbc25b63